functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/BuildConfigurator.java

```diff
@@ -118,6 +120,16 @@ public BuildConfigurator ignoreBuildFailure() {
         return this;
     }
 
+    public BuildConfigurator skipTests(boolean skipTests) {
+        this.skipTests = skipTests;
+        return this;
+    }
+
+    public BuildConfigurator withMavenLog() {
```

Comment:
It's not really needed. Maven log is always in the case of how we use embedded maven, so simple method for retrieving it from the build is enough. This is making it overcomplicated considering how it's done underneath.